packages/core/src/servient.ts

```diff
@@ -30,6 +30,9 @@ export default class Servient {
     private things: Map<string, ExposedThing> = new Map<string, ExposedThing>();
     private credentialStore: Map<string, Array<unknown>> = new Map<string, Array<unknown>>();
 
+    #startedWoT?: typeof WoT;
```

Comment:
Maybe a field name like `wotInstance` could be a bit clearer here?

Reply:
Sure 👍
I will wait till we are fine with the proposed changes and apply the change...

Reply:
fixed with https://github.com/eclipse-thingweb/node-wot/pull/1195/commits/aa1537187cad8789907972262b7ef15c2f55fc0d